testing/inner_default_resource_test.go

```diff
@@ -0,0 +1,34 @@
+/*
+Copyright 2019 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package testing
+
+import "testing"
+
+func TestInnerDefaultResource_Validate(t *testing.T) {
+	r := InnerDefaultResource{}
+	if err := r.Validate(); err != nil {
+		t.Fatalf("Expected no validation errors. Actual %v", err)
```

Comment:
`Errorf()` would suffice, but at this point it doesn't matter much.

Reply:
When should I prefer `Errorf` to `Fatalf`? Here it clearly doesn't matter as this is the end of the method anyway. I tend to default to `Fatalf` unless I think I can get more information by continuing the test.

Reply:
`Fatalf` kills the current go routine and in general is more intrusive. Why do more, if you don't have to?
I tend to use fatalf iff the rest of test is rendered useless because of this failure.
E.g. getting back an error on some Get call, renders checking the contents of the returned object useless.